Cargo.toml

```diff
@@ -9,15 +9,15 @@ repository = "https://github.com/buehler/zitadel-rust"
 keywords = ["ZITADEL", "gRPC", "clients", "authentication", "oidc"]
 categories = ["authentication", "api-bindings"]
 include = [
-    "build.rs",
```

Comment:
I ran this through taplo. It may or may not be wanted. feel free to request a revert =D

Reply:
What is taplo? as far as I know, you'll need the build.rs file to create the grpc files locally right?

Reply:
ah, I get it. no worries, the whitespace does not bother me much. I misread and thought you removed the build.rs file :)